src/commands/cmd_zset.cc

```diff
@@ -1109,9 +1109,9 @@ class CommandZMScore : public Commander {
       for (const auto &member : members) {
         auto iter = mscores.find(member.ToString());
         if (iter == mscores.end()) {
-          values.emplace_back(conn->NilString());
+          values.emplace_back("");
         } else {
-          values.emplace_back(conn->Double(iter->second));
+          values.emplace_back(util::Float2String(iter->second));
```

Comment:
We can replace `conn->MultiBulkString(values);` in 1118th line to `Array(..)` instead of changing here.

Reply:
I think it is better than Array(values), because conn->MultiBulkString(values) converge protocol mean；and values only expression values；  If use Array(values), then values include some protocol meaning，it is incomplete

Reply:
That's correct to use `redis::Array` instead of `MultiBulkString` because it needs to return the double type in RESP3.

Reply:
OK，get it，I will update